spec/Overview.html

```diff
@@ -1066,43 +1064,105 @@ <h3>CryptoKey interface members</h3>
+            </li>
+            <li>
+              Initialize the <a>[[\usages]]</a> internal slot of <var>value</var> to the
+              [= sub-deserialization =] of <var>serialized</var>.[[\Usages]].
+            </li>
+            <li>
+              If the <a>[[\type]]</a> internal slot of <var>value</var> is {{KeyType/"secret"}},
+              then let <var>importedKey</var> be the result of performing the import key
+              operation specified by the <a>[[\algorithm]]</a> internal slot of <var>value</var>,
+              using {{KeyFormat/"raw"}} as <var>format</var>, the [= sub-deserialization =] of
+              <var>serialized</var>.[[\SecretKey]] as <var>keyData</var>, and the <a>[[\algorithm]]</a>,
+              <a>[[\extractable]]</a> and <a>[[\usages]]</a> internal slots of <var>value</var>
+              as <var>algorithm</var>, <var>extractable</var> and <var>usages</var>, respectively.
+            </li>
+            <li>
+              If the <a>[[\type]]</a> internal slot of <var>value</var> is {{KeyType/"private"}}, then
+              then let <var>importedKey</var> be the result of performing the import key
+              operation specified by the <a>[[\algorithm]]</a> internal slot of <var>value</var>,
+              using {{KeyFormat/"pkcs8"}} as <var>format</var>, the [= sub-deserialization =] of
+              <var>serialized</var>.[[\PrivateKey]] as <var>keyData</var>, and the <a>[[\algorithm]]</a>,
+              <a>[[\extractable]]</a> and <a>[[\usages]]</a> internal slots of <var>value</var>
+              as <var>algorithm</var>, <var>extractable</var> and <var>usages</var>, respectively.
+            </li>
+            <li>
+              If the <a>[[\type]]</a> internal slot of <var>value</var> is {{KeyType/"public"}}, then
+              then let <var>importedKey</var> be the result of performing the import key
+              operation specified by the <a>[[\algorithm]]</a> internal slot of <var>value</var>,
+              using {{KeyFormat/"spki"}} as <var>format</var>, the [= sub-deserialization =] of
+              <var>serialized</var>.[[\PublicKey]] as <var>keyData</var>, and the <a>[[\algorithm]]</a>,
+              <a>[[\extractable]]</a> and <a>[[\usages]]</a> internal slots of <var>value</var>
+              as <var>algorithm</var>, <var>extractable</var> and <var>usages</var>, respectively.
+            </li>
+            <li>
+              Initialize the <a>[[\handle]]</a> internal slot of <var>value</var> to refer to the
+              cryptographic key data represented by the <a>[[\handle]]</a> internal slot of <var>importedKey</var>.
             </li>
           </ol>
           <div class=note>
-            When performing the structured clone algorithm in
-            order to serialize a {{CryptoKey}} object, implementations must not allow the
+            When deserializing a serialized {{CryptoKey}} object, implementations must not allow the
```

Comment:
You should not have requirements in notes (i.e., no must). Just statements of fact. And you shouldn't have duplicate requirements either so please don't move it out of a note either. Stating something again is good though.